### DISCUSSION TITLE
Transfer of Claim blocks

### DISCUSSION THREAD

**Discussion by SvenjaReissaus** [GriefPrevention/GriefPrevention#1005] [Ideas]
**As** a player AND/OR admin
**I want** to be able to transfer mine (OR other's, in case of admin) player's claim blocks.
**Because** It would make it easier for players that play on teams to have a unified control over their lands. And, probably provide another manner of "Trade" for such.

**Is your feature request related to a problem? Please describe.**
No. It would just make some kind of gameplays, easier.

**Describe alternatives you've considered**
Ever heard of Factions? Yes. We are not getting into that.

This could also be the start of a new PR, I have the thought half planned in my head as to _how to_. But considering in "asking" for approval as if such feature would be merged into the main branch of the project. Since again, would make it a little easier for players that go on teams (Yes, like me).

Any info regarding if this feature is planned, or intended, and a status update about it would be appreciated, as well as indication as if to submit said PR.

**Comment by RoboMWM** [comment]
Think this is a duplicate, but this can be easily done in an addon. As I mentioned in wherever that duplicate issue is, my fork of gprealestate performs this transaction.

**Comment by SvenjaReissaus** [comment]
Is said addon this one? https://www.spigotmc.org/resources/gp-realestate.11422/
If so, it just seems like an extended version of /sellclaimblocks, but rather than selling to server, sells to players. Which _could_ somewhat mask the feature, but I was aiming more towards a clean transfership, rather than a sell-buy solution.
I would believe it can be simple (can't think of a way of making it complicated) as to incorporated in the main plugin behind a permission node, rather to have a whole addon to bloat the server just to have said feature in place. But let me know your thoughts!

**Comment by RoboMWM** [comment]
No I did not publish my fork.

**Comment by RoboMWM** [comment]
it's on my GitHub profile.
